crmsh/ui_cluster.py

```diff
@@ -74,6 +74,7 @@ class Cluster(command.UI):
     - Perform other callouts/cluster-wide devops operations
     '''
     name = "cluster"
+    ENABLE_SERVICES = ("pacemaker.service", "corosync-qdevice.service")
```

Comment:
I wonder we have to add the condition if corosync-qdevice is set in corosync.conf, or not. Blindly manage qdevice.service is a little too much for the cluster without qdevice.

Reply:
Never mind @zzhou1 
Inside function `utils.enable_service`, the logic will check if the service is available